app/page.tsx

```diff
@@ -100,7 +29,40 @@ const PARTNERS = [
 export default function Home() {
 	return (
 		<>
-			<RootLayout>
+			<Header expanded={true} className={styles.landingHeader}>
```

Comment:
For a11y purpose, this could be a nav element (we can keep naming it header)

Reply:
I'm not sure to understand what you mean, i see why it's good to have a nav for accessibility but how can i have a nav named header ? Should i just put this component in a nav tag ?

Reply:
Tu peux garder le composant `Header` mais dans le composant il faut une balise `nav`